### PR TITLE
Added ability to supply parsed transformation as array.

### PR DESCRIPTION
**Pull request by vail130**

### CODE REVIEW

**Comment by jonathan-ostrander** [comment]
LGTM

File: composable.js [hunk 1]
```diff
@@ -287,29 +287,52 @@
...
+            }
+
+            args = transformationInput.slice(1);
+            if (arrayTransformations.hasOwnProperty(command)) {
+                subCommand = this.getTransformation(args);
+                if (subCommand) {
+                    args = [subCommand];
+                }
+            }
+        } else {
```
Comment (jonathan-ostrander) [inline composable.js:305]: Maybe split this into a private method, but it doesn't seem necessary unless more additions to this are planned.

**Comment by vail130** [comment]
@OstrichProjects updated

File: test/spec.js [hunk 2]
```diff
@@ -104,6 +105,20 @@ describe('Composable', function () {
...
+                a: [
+                    ['window'],
+                    ['getProperties', 'testData2.stringWithComma'],
+                    ['split', ','],
+                    ['map', 'trim']
+                ]
+            });
+            expect(extractedData.a).toEqual(['Hello', 'and welcome!']);
+        })
+    });
```
Comment (ssttuu) [inline test/spec.js:132]: can we either add a test or modify the above to use mixed array and string transformations?  e.g.

```
var extractedData = Composable({
    a: [
        'window',
        'getProperties:testData2.stringWithComma',
        ['split', ','],
        'map:trim'
]
});
```

That should work, right?

Comment (vail130) [inline test/spec.js:132]: Yes it should. I'll add a test.

**Comment by ssttuu** [comment]
LGTM

**Pull request merged**